packages/deno/src/integrations/globalhandlers.ts

```diff
@@ -110,7 +110,7 @@ function installGlobalUnhandledRejectionHandler(client: Client): void {
       originalException: error,
       mechanism: {
         handled: false,
-        type: 'unhandledrejection',
+        type: 'auto.deno.global_handlers.unhandledrejection',
```

Comment:
I don't know if we should separate this word here.

```suggestion
        type: 'auto.deno.global_handlers.unhandled_rejection',
```